### PR TITLE
Feat: lesson1

### PR DESCRIPTION
**Pull request by yuka830**
[Lesson1 ](https://github.com/kenmori/handsonFrontend/blob/master/work/markup/1.md#1%E3%81%93%E3%81%AEdom%E3%82%92html%E5%86%85%E3%81%AEul%E3%81%AE%E4%B8%AD%E3%81%AB%E5%B7%AE%E3%81%97%E8%BE%BC%E3%82%93%E3%81%A7%E3%81%8F%E3%81%A0%E3%81%95%E3%81%84)

[CodeSandBox](https://codesandbox.io/s/sharp-https-u8obf)
[Last commit](https://github.com/yuka830/js-lessons/pull/2/commits/68a2cb632bb3c1e19ccfa69edaa2c0f797b3b966)

Created new li with createElement and textContent, and appended it with id and appendChild.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+const lists = document.getElementById("js-lists");
+const newItem = document.createElement("li");
+newItem.textContent = "これです";
+lists.appendChild(newItem);
```
Comment (csakyo) [inline main.js:4]: It looks good!! I approve this!

[IMO]
I think you should use the same naming method for the Id name and constant name.
For example, 'js-lists' is the kebab case, 'newItem' is the camel case. And there is a snake case as well. If you are interesting in it, please search about it.

Comment (ghost) [reply main.js:4]: [IMO]
IDs and classes that are the starting point of JavaScript are prefixed with `.js-` to clearly indicate that JavaScript is involved.（I'm sorry, I could only find useful articles in Japanese.）[Reference article](https://zenn.dev/haniwaman/articles/bf392f397c8db7341881#javascript%E3%81%A8%E7%B4%90%E3%81%A5%E3%81%8F%E8%A6%81%E7%B4%A0%E3%81%AFjs-%E3%81%AE%E6%8E%A5%E9%A0%AD%E8%BE%9E)
I thought this way of writing would be fine

**Review by ghost — APPROVED** [review]

**Comment by kenmori** [comment]
I offen use
`js-***`  is js event trigger. (no assign with style).
`hogeHoge` is also used variable name in js.

I rarely seen `hoge-hoge` or `hoge_hoge` in js file. It is used class name. but I don't use those (not use in js)

Nowadays, class names are also used in camelcase in the context of jsx(check it if you dont know yet)

**Comment by yuka830** [comment]
Thank you guys.It was good opportunity for me to think about naming.This time I'm going to marge them without any changes.

**Pull request merged**